modules/sdk-coin-xlm/test/unit/xlm.ts

```diff
@@ -314,12 +314,12 @@ describe('XLM:', function () {
     // This key pair is the decoded version of the userKeychain above
     const rootKeychain = {
       pub: '37c6be1c677873e42a90f3d0ed3b0e4f3ba8591f285769d8c922f7024ae0c009',
-      prv: 'c034ca796a145e79acfac9bc9d97e4d3aaa42bfbe648b46bb1f3da794d7765fb',
+      prv: 'c034ca796a145e79acfac9bc9d97e4d3aaa42bfbe648b46bb1f3da794d7765fb37c6be1c677873e42a90f3d0ed3b0e4f3ba8591f285769d8c922f7024ae0c009',
     };
     // This key pair is the decoded version of the backupKeychain above
     const backupRootKeychain = {
       pub: 'b63de5ad57cc75b256483bc1c1de82689497eaf27a930896e7733f418acf9eca',
-      prv: '35a98c286af2c36283adcd47e92cc221b1cd24eeded852a21abfbbc65e98eb8a',
+      prv: '35a98c286af2c36283adcd47e92cc221b1cd24eeded852a21abfbbc65e98eb8ab63de5ad57cc75b256483bc1c1de82689497eaf27a930896e7733f418acf9eca',
     };
```

Comment:
Instead of changing these, would it make sense to create a new keypair in the new format? That way we still have tests for the old format.

Reply:
Hmm, I changed these as I realized that root prvs are represented in `prv + pub` format, so I think that the original format of these test keychains actually would not occur

Reply:
It could be that we used to support this format but then changed it later, which would mean some old wallets would have that old format. I am not actually sure, but if the tests passed with the old format, I'd expect that it should still work? 

Are the old formats still valid for the tests they are used for or do they error out?

Reply:
So these tests and this test data I added in the initial root key PR for XLM here: https://github.com/BitGo/BitGoJS/pull/4638

Before this, no XLM wallet was able to be created using root keys. I reason I updated these keys was because I realized that root keys generated in the OVC, specifically the private key part, are given in an encrypted version of the `prv + pub` 64-byte format, rather than that old 32-byte `prv` only one.

The old format does still pass the tests, since in the code the place they are used is calling a `slice(0, 64)` to extract the first 32-bytes of the hex string anyway.